src/main/webapp/dev/AdminHomeJsTest.es6

```diff
@@ -1,77 +1,85 @@
-'use strict';
+/* global
+addInstructorAjax:false, enableAddInstructorForm:true, createRowForResultTable:false
+addInstructorByAjaxRecursively:true, addInstructorFromFirstFormByAjax:false, addInstructorFromSecondFormByAjax:false
+*/
 
 QUnit.module('AdminHome.js');
 
-QUnit.assert.contains = function(context, toIdentify, message) {
-    var actual = context.indexOf(toIdentify) > -1;
+QUnit.assert.contains = function (context, toIdentify, message) {
+    const actual = context.indexOf(toIdentify) > -1;
     this.pushResult({
         result: actual,
-        actual: actual,
+        actual,
         expected: toIdentify,
-        message: message
+        message,
     });
 };
 
-QUnit.test('createRowForResultTable(shortName, name, email, institution, isSuccess, status)', function(assert) {
-    var boolIndex = 4;
-    var successClass = 'success';
-    var failureClass = 'danger';
+QUnit.test('createRowForResultTable(shortName, name, email, institution, isSuccess, status)', (assert) => {
+    const boolIndex = 4;
+    const successClass = 'success';
+    const failureClass = 'danger';
     function testCreateRowForResultTable(isSuccess) {
-        var testProperties = ['test', 'testName', 'testMail', 'testInstitution', isSuccess, 'testStatus'];
-        var result = createRowForResultTable.apply(null, testProperties);
-        var expected = testProperties.slice();  // deep clone testProperties
+        const testProperties = ['test', 'testName', 'testMail', 'testInstitution', isSuccess, 'testStatus'];
+        const result = createRowForResultTable(...testProperties);
+        const expected = testProperties.slice();  // deep clone testProperties
         expected[boolIndex] = isSuccess ? successClass : failureClass;
-        expected.forEach(function(property) {
-            assert.contains(result, property, 'should contain ' + property);
+        expected.forEach((property) => {
+            assert.contains(result, property, `should contain ${property}`);
         });
     }
     [true, false].forEach(testCreateRowForResultTable);
 });
 
-var addInstructorDetailsSingleLine = '<textarea id="addInstructorDetailsSingleLine"></textarea>';
-var addInstructorResultTable = '<table id="addInstructorResultTable"><tbody></tbody></table>';
```

Comment:
Yup, unused. Normally we'll list it under `/* exported */` just in case, but since this is test code it's bound to be unused at all.